src/main/java/teammates/common/datatransfer/attributes/AdminEmailAttributes.java

```diff
@@ -151,4 +167,43 @@ public String getFirstAddressReceiver() {
     public String getFirstGroupReceiver() {
         return getGroupReceiver().get(0);
     }
+
+    public static class Builder {
+        private final AdminEmailAttributes adminEmailAttributes;
+
+        public Builder(String subject, List<String> addressReceiver, List<String> groupReceiver,
+                       Text content, Date sendDate) {
+            adminEmailAttributes = new AdminEmailAttributes();
+            adminEmailAttributes.addressReceiver = addressReceiver;
+            adminEmailAttributes.groupReceiver = groupReceiver;
+            adminEmailAttributes.subject = subject;
+            adminEmailAttributes.content = content;
+            adminEmailAttributes.sendDate = sendDate;
+        }
+
+        public Builder withCreateDate(Date createDate) {
+            if (createDate != null) {
+                adminEmailAttributes.createDate = createDate;
+            }
+            return this;
+        }
+
+        public Builder withEmailId(String emailId) {
+            if (emailId != null) {
+                adminEmailAttributes.emailId = emailId;
+            }
+            return this;
+        }
+
+        public Builder withIsInTrashBin(Boolean isInTrashBin) {
```

Comment:
Use just boolean here, not wrapper.

Reply:
We cannot check for null with the primitive boolean type. 
Therefore, .withIsInTrashBin(null) is a compilation error.
I could ommit the test for null here, but I don't think this is a good idea. What do you think?

Reply:
Why do you need to test for `null` if passing `null` to this method is a compile-time error in the first place?